client.go

```diff
@@ -21,6 +22,8 @@ const statusSuccess = "SUCCESS"
 // DefaultTTL The minimum and the default is 300 seconds.
 const DefaultTTL = "300"
 
+var retryableCodes = []int{429, 502, 503, 504}
```

Comment:
The 50x statuses are not retryable by definition: they are server errors.

Reply:
In certain circumstances retrying server errors make sense. In this situation Porkbun returns a 503 when you're being rate limited so we want to retry it. But I'm good making the changes, will push a commit shortly.